### PR TITLE
Add delta > 0 validation to Huber loss

### PR DESCRIPTION
**Pull request by kaivalya-cyber**
The Huber loss docs describe delta as the transition point from quadratic to linear. delta <= 0 produces nonsensical results (division by zero for delta=0). This adds a ValueError at construction time to catch invalid configurations.

### CODE REVIEW

**Review by dmiltr3 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

We appreciate your effort to add validation to the `Huber` loss parameter `delta`. However, there are a few important points regarding this Pull Request:

1. **Legacy Codebase**: This PR targets `tensorflow/python/keras/losses.py`, which is part of the legacy Keras implementation embedded in TensorFlow. The Keras codebase has been refactored and is now actively maintained as a standalone multi-backend framework in the `keras-team/keras` repository. New features and bug fixes should target that repository to benefit all users.

2. **Backend Agnosticism**: In the active multi-backend Keras, `tensor_util.is_tf_type` is not appropriate as it is specific to the TensorFlow backend. The validation should support multiple backends (TensorFlow, JAX, PyTorch).

3. **Recommended Implementation**: If you wish to implement this in the active codebase (e.g., in `keras/src/losses/losses.py`), here is the recommended approach for `Huber.__init__`:

```python
        if not ops.is_tensor(delta) and delta <= 0:
            raise ValueError("delta must be greater than 0.")
```
*(Note: `ops` refers to `keras.ops`)*

4. **Testing**: You should also add unit tests to verify this validation. Here is an example test case that can be added to `keras/src/losses/losses_test.py`:

```python
    def test_invalid_delta(self):
        with self.assertRaisesRegex(ValueError, "delta must be greater than 0"):
            losses.Huber(delta=0.0)
        with self.assertRaisesRegex(ValueError, "delta must be greater than 0"):
            losses.Huber(delta=-1.0)
```

Please consider closing this PR and opening a new one against the active Keras repository with these adaptations.

**Comment by kaivalya-cyber** [comment]
@dmiltr3 thanks for the guidance — closing this. Will re-implement against
the active keras-team/keras repo using the backend-agnostic pattern you
suggested (`ops.is_tensor(delta)`). Appreciate the detailed review!
